packages/create-audius-app/index.test.ts

```diff
@@ -0,0 +1,167 @@
+import execa from 'execa'
+import fs from 'fs-extra'
+import path from 'path'
+import { useTempDir } from './test/use-temp-dir'
+import { describe, it, expect } from 'vitest'
+import {
+  projectFilesShouldExist,
+  projectFilesShouldNotExist
+} from './test/utils'
+
+const cli = require.resolve('create-audius-app/dist/index.js')
```

Comment:
Interesting to test built stuff but suppose that's the move. There's a way to test with transpiration right?

Reply:
And I've set up the turbo pipeline so `test` requires `build`! What do you mean test with transpiration?